docs/configuration_file.md

```diff
@@ -139,6 +139,8 @@ Some options are also mandatory.
 
 * `reactor_config_secret` (*optional*, `string`) — name of Kubernetes secret holding [atomic-reactor configuration file](https://github.com/projectatomic/atomic-reactor/blob/master/docs/config.md)
 
+* `reactor_config_map` (*optional*, `string`) — name of Openshift's ConfigMap holding reactor_config
```

Comment:
ConfigMap is a Kubernetes thing, not OpenShift-specific, so let's say "name of Kubernetes ConfigMap holding atomic-reactor configuration file". Also, can we make that a link as for reactor_config_secret?

Reply:
sure